providers/implementations/keymgmt/ec_kmgmt.c

```diff
@@ -358,8 +358,9 @@ int ec_export(void *keydata, int selection, OSSL_CALLBACK *param_cb,
 
     if ((selection & OSSL_KEYMGMT_SELECT_DOMAIN_PARAMETERS) != 0) {
         bnctx = BN_CTX_new_ex(ec_key_get_libctx(ec));
+        if (bnctx == NULL)
+            goto err;
         BN_CTX_start(bnctx);
-        ok = ok && (bnctx != NULL);
```

Comment:
`ok` is not sent to false anymore in the error case. `err` is probably a confusing name for the label since it's always executed, not only in case of error.

Reply:
Thanks for spotting it..